resources/views/includes/header.blade.php

```diff
@@ -18,8 +18,12 @@
             </div>
             <div class="dark-light-buttons">
                 <span class="mode-text">@lang('main.color-mode')</span>
-                <button class="lightButton"></button>
-                <button class="darkButton"></button>
+                <button class="lightButton">
+                    <span class="emptyBtn">.</span>
```

Comment:
@hambardzumianvahag as I understand **emptyBtn** should be **display none**. So what is purpose of this?